goal_src/engine/game/game-info-h.gc

```diff
@@ -187,9 +187,9 @@
 (define-extern *game-info* game-info)
 (when (or (not *game-info*) (zero? *game-info*))
   (let ((temp (new 'static 'game-info :mode 'debug :current-continue #f)))
-    (set! (-> temp fuel-cell-time) (new 'global 'boxed-array int64 116))
-    (set! (-> temp enter-level-time) (new 'global 'boxed-array int64 32))
-    (set! (-> temp in-level-time) (new 'global 'boxed-array int64 32))
+    (set! (-> temp fuel-cell-time) (new 'global 'boxed-array time-frame 116))
```

Comment:
It probably doesn't matter, but I think this is wrong - I think the compiler will make the `content-type` field of this array be `time-frame` because there's a runtime `time-frame` type.